lib/cog/commands/alias.ex

```diff
@@ -6,31 +6,35 @@ defmodule Cog.Commands.Alias do
   @moduledoc """
   Manages aliases
 
-  ## Subcommands
-  * new <alias-name> <alias-definition> -- creates a new alias visible to the creating user.
-  * mv <alias-name> <site | user>:[new-alias-name] -- moves aliases between user and site visibility. Optionally renames aliases.
-  * rm <alias-name> -- Removes aliases
-  * ls [pattern] -- Returns the list of aliases optionally filtered by pattern. Pattern support basic wildcards with '*'.
+  USAGE:
+    alias <subcommand>
 
-  ## Example
 
-      @bot #{Cog.embedded_bundle}:alias new my-awesome-alias "echo \"My Awesome Alias\""
-      > user:my-awesome-alias has been created
+  SUBCOMMANDS:
+    new <alias-name> <alias-definition>             creates a new alias visible to the creating user.
+    mv <alias-name> <site | user>:[new-alias-name]  moves aliases between user and site visibility. Optionally renames aliases.
```

Comment:
I'm trying to not change the content of the usage info right now. I'm just reformatting. Honestly I think all of this should be removed from here and moved into the subcommand module.

But to answer your question, yes. `mv` moves the alias to either the `site` or `user` visibility and can optionally change the name of the alias. So if you are moving visibility and changing the name it would require the colon. But I agree, I don't think it's particularly clear.